app/controllers/portal/tax_returns_controller.rb

```diff
@@ -0,0 +1,79 @@
+module Portal
```

Comment:
This looks like a really big addition, but it's actually just moving an existing file under the Portal namespace.